server/utils/validate.js

```diff
@@ -22,6 +22,18 @@ const validate = {
       .isString()
       .withMessage('Your password is invalid'),
   ],
+  newEntry: [
+    check('title')
+      .isString()
+      .withMessage('Title should be a string'),
+    check('content')
+      .isString()
+      .withMessage('Content should be a string'),
+    check('is_favorite')
+      .isBoolean()
+      .withMessage('isFavorite property of a story should be boolean'),
+  ],
+  isNumber: number => !Number.isNaN(Number(number)),
 };
```

Comment:
You've been using `classes` which have been quite nice, why not use `classes` in this file?

Reply:
Nice observation. I will make immediate amends for consistency